File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Three/C++/prog5.cpp
#include<iostream>

int main() {
	int num, sum;
	std::cout << "Enter num : ";
	std::cin >> num;
	for(int i = 2; i <= num; i++) {
		sum = 0;
		for(int j = 1; j <= i/2 ; j++) {
			if(i % j == 0)
				sum += j;
		}
		if(sum == i)
			std::cout << i << " ";
	}
	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Three/Python/prog5.py
s = 0
num = int(input("num : "));
for i in range(2, num + 1):
    s = 0
    for j in range(1, (int(i / 2) + 1)):
            if(i % j == 0):
                s += j
    if(s == i):
        print(s, end = " ")
